src/main/java/io/vertx/mqtt/MqttServerOptions.java

```diff
@@ -252,10 +252,29 @@ public MqttServerOptions setTimeoutOnConnect(int timeoutOnConnect) {
     return this;
   }
 
+  /**
+   * Do the same thing as {@link MqttClientOptions#setKeepAliveTimeSeconds(int)}. Use it instead.
```

Comment:
Do the same thing as `{@link MqttClientOptions#setTimeoutOnConnect(int)}`. Use it instead.